src/lib/room.js

```diff
@@ -38,6 +38,11 @@ export const onChatClose = async () => {
 		postChatUrl: null,
 		room: null,
 	});
+	if (isWebView()) {
+		// Hide Print
+		store.setState({ hidePrint: true });
+		parentCall('hidePrint');
+	}
```

Comment:
@AlexanderKanakis This is just setting `hidePrint` flag when the widget is running in the webview. But I think what we want to achieve is when `hidePrint` is true we want to not show the `Save Transcript` button. Is that correct @ear-dev?

Reply:
@Shailesh351 Do we want not to show `Save Transcript` in any case?

Reply:
we should abide by showPrint or hidePrint in both web and mobile.  We want the integration script to decide.  We should just be checking for true or false and responding accordingly.  This PR should be web/mobile agnostic.